matrix/utils.py

```diff
@@ -227,6 +227,8 @@ def translate_ansi_colors(entity):
 
     '''
     log.info("Running crash dump")
+    if controller:
+        model_name = "{}:{}".format(controller, model_name)
```

Comment:
I'm a little worried here because I don't know who calls `crashdump` with a `model_name`.  Is it possible the caller would already prefix the model name with `controller:`?  If so, perhaps the conditional here should be:

```python
if controller and (controller not in model_name):
```

That may not be robust enough given i name my controllers things like `g`.  So maybe check for a `.split(:)` before blindly appending the controller?

Or this may be moot if you know crashdump callers can't use a controller prefix when specifying the model_name.  Just food for thought.